test/test.js

```diff
@@ -1,12 +1,68 @@
-var request = require('supertest');
+var supertest = require('supertest');
+var db = require('./db');
+
+var request = function (path) {
+    return supertest('http://localhost:6969')
+    .get(path)
+    .expect(200);
+}
+
+describe.skipIfImpl = function(impls, title, fn) {
+  var currentImpl = process.env["EBOSHI_API_SHOOTOUT_CURRENT_IMPL"];
+  if(impls.indexOf(currentImpl) !== -1) {
+      return this.skip(title, fn);
+  } else {
+      return this(title, fn);
+  };
+};
 
 describe('/api/test', function() {
+    it('returns "Hello world"', function (done) {
+        request('/api/test').expect('Hello world', done);
+    });
+});
 
-    it('should return Hello world', function (done) {
-        request('http://localhost:6969')
-        .get('/api/test')
-        .expect(200)
-        .expect('Hello world', done);
+describe.skipIfImpl(['./elixir_phoenix', './node_express', './ruby_sinatra'], '/api/clients', function() {
+    before(function(done) { db.bootstrap(done); });
+
+    before(function(done) {
+        db.query(
+            "INSERT INTO clients SET " +
+            "  name='Bot and Rose Design', " +
+            "  address='625 NW Everett St', " +
+            "  city='Portland', " +
+            "  state='OR', " +
+            "  zip='97209', " +
+            "  country='USA', " +
+            "  email='info@botandrose.com', " +
+            "  contact='Michael Gubitosa', " +
+            "  phone='(503) 662-2712', " +
+            "  created_at='2006-06-25T14:08:31Z', " +
+            "  updated_at='2015-08-29T09:58:23Z';",
```

Comment:
yeah, unfortunately it becomes a pain in the ass with Travis if it is not specified as a node project I think (as we are setting it to elixir).  if it is you can easily set the node version via the .yml file (they use nvm).  local node management is easy with n or nvm.

It would be nice to run this on node 4 thought to get the ES6 goodies.